include/rxterm/terminal.hpp

```diff
@@ -27,8 +31,35 @@ struct VirtualTerminal {
     std::flush(std::cout);
     return {next};
   }
+
+#ifdef __WIN32
+  static std::once_flag initInstanceFlag;
+  static DWORD initWindowsTerminal() {
+#ifndef ENABLE_VIRTUAL_TERMINAL_PROCESSING
+#define ENABLE_VIRTUAL_TERMINAL_PROCESSING 0x0004
+#endif
+  // Set output mode to handle virtual terminal sequences
+  HANDLE hOut = GetStdHandle(STD_OUTPUT_HANDLE);
```

Comment:
Where are GetStdHandle and GetConsoleMode comming from?

Reply:
Yes, already correct this in new commit (#include <windows.h>"). And tested it on MSVS 2019 complier. It's need to replace string literals with "\e..." to "\033...". Do new PR?

Reply:
>  It's need to replace string literals with "\e..." to "\033...".
Why? Both are equivalent: https://godbolt.org/z/6bMbbd

The library just seemed to work out of the box after calling `system(" ")`
https://github.com/LoopPerfect/rxterm/issues/4#issuecomment-706785590

Reply:
Is maybe SetConsoleMode behaving differently to system(" ") ?

Reply:
Also in the past someone claimed that it should be changed to '\x1B' (which is identical to '\e' ?!)
https://github.com/LoopPerfect/rxterm/pull/9/files

Unlike '\x1B', `\033` is definetly a different escape code than '\e', however it still works under linux

Reply:
> Also in the past someone claimed that it should be changed to '\x1B' (which is identical to '\e' ?!)
> Unlike '\x1B', `\033` is definetly a different escape code than '\e', however it still works under linux

_Hex_ \x1B = _Oct_ \033 = _Dec_ 27 = \e 
But MSVS compiler does not recognized "\e" literal, only \x1B or \033 variant. For GCC and Clang compilers all this expressions are identical

Reply:
> The library just seemed to work out of the box after calling `system(" ")`
Yes, it works, but not compiled with MSVC by "\e" errors. 

> Is maybe SetConsoleMode behaving differently to system(" ") ?
SetConsoleMode is documented kind for system console to emulate VT100 terminal, but I can't explain system(" ") magic :). Call system(" ") from code is addition call system command interpreter. On my opinion, this is not "clean code" and not documented behavior. I'm not sure, that this hack will normal worked in future version Windows Terminal or another shell replacers or IDE debug console windows.